packages/core/src/libraries/sign-in-experience/index.test.ts

```diff
@@ -18,34 +18,42 @@ const { mockEsm } = createMockUtils(jest);
 
 const allCustomLanguageTags: LanguageTag[] = [];
 
-const { findAllCustomLanguageTags } = mockEsm('#src/queries/custom-phrase.js', () => ({
```

Comment:
Why removing this `mockEsm`?

Reply:
https://github.com/logto-io/logto/pull/2867

unit test pattern changed